src/Security/MemberAuthenticator/Authenticator.php

```diff
@@ -112,9 +134,8 @@ protected static function authenticate_member($data, $form, &$success)
      *
      * @param array $data
      * @param Member $member
-     * @param bool $success
      */
-    protected static function record_login_attempt($data, $member, $success)
+    protected function recordLoginAttempt($data, $member, $success)
     {
         if (!Security::config()->login_recording) {
```

Comment:
Wouldn't it be better to have this if before the call, instead of inside the method?